### PR TITLE
Fixes #28228 - fix linting errors

### PR DESCRIPTION
**Pull request by Ron-Lavi**
Due to prettier update, some of the linting rules have changed,
this PR fixes them, and updates `package.json` with the latest prettier version.

### CODE REVIEW

**Comment by Ron-Lavi** [comment]
snapshots errors fix here: #7162

**Comment by Ron-Lavi** [comment]
update the the failing tests snapshots here as well.

**Review by amirfefer — DISMISSED** [review]
Thanks @laviro  LGTM :+1: 
foreman failure seems unrelated.

File: webpack/assets/javascripts/react_app/components/Editor/components/__tests__/__snapshots__/EditorSettings.test.js.snap [hunk 1]
```diff
@@ -18,7 +18,7 @@ exports[`EditorSettings renders EditorSettings 1`] = `
         >
           Syntax
         </div>
-        <Uncontrolled(Dropdown)
+        <ForwardRef
```
Comment (amirfefer) [inline webpack/assets/javascripts/react_app/components/Editor/components/__tests__/__snapshots__/EditorSettings.test.js.snap]: please notice that this snap is already fixed in [your other PR](https://github.com/theforeman/foreman/pull/7162/files#diff-9ff6832b207b4729b975babe0669a85a)

Comment (sharvit) [reply webpack/assets/javascripts/react_app/components/Editor/components/__tests__/__snapshots__/EditorSettings.test.js.snap]: Not relevant in this PR

**Review by sharvit — CHANGES_REQUESTED** [review]

**Comment by Ron-Lavi** [comment]
Removed the snapshots fix from this PR to #7165

File: package.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
     "jest-cli": "^23.6.0",
     "jest-prop-type-error": "^1.1.0",
     "node-sass": "^4.5.0",
-    "prettier": "^1.18.0",
+    "prettier": "^1.19.1",
```
Comment (tbrisker) [inline package.json:72]: Should we use an exact version here so it doesn't break again when a new version gets released?

Comment (Ron-Lavi) [reply package.json:72]: IMO we benefit from their upgrades and if we will forget to upgrade from time to time,
the changes might be too big and hard to review.

**Comment by sharvit** [comment]
Can you please rebase so we will be sure Travis is green from now on?

**Comment by Ron-Lavi** [comment]
Rebased @sharvit

**Comment by Ron-Lavi** [comment]
Everything is green :green_heart:

**Review by amirfefer — APPROVED** [review]
Thanks @laviro !

**Review by sharvit — APPROVED** [review]
Thanks @laviro

**Comment by ezr-ondrej** [comment]
We need ack from packaging, @tbrisker ?

**Review by ekohl — APPROVED** [review]
prettier is skipped in packaging so :+1: from that side.

**Comment by ezr-ondrej** [comment]
@amirfefer, @sharvit wanna go ahead and push the button? :slightly_smiling_face:

**Comment by sharvit** [comment]
> @amirfefer, @sharvit wanna go ahead and push the button?

Merged Thanks @laviro @amirfefer @ekohl @ezr-ondrej 

@amirfefer Sorry from stilling from you ;)

**Pull request merged**